storage/cloudspanner/map_storage_test.go

```diff
@@ -33,7 +45,11 @@ var cloudDBPath = flag.String("test_cloud_spanner_database", "", "eg: projects/m
 func GetTestDB(ctx context.Context, t *testing.T) *spanner.Client {
 	t.Helper()
 	if *cloudDBPath == "" {
-		t.Skip("-test_cloud_spanner_database flag is unset")
+		ddl, err := readDDL()
```

Comment:
Worth extending the flag description string above (L43) to mention that leaving empty means use the in-mem test DB?

Reply:
Added `:memory:` option